server/src/api.ts

```diff
@@ -3,37 +3,46 @@ import {GameModel} from 'common/models/game-model'
 import {createRequire} from 'module'
 import root from './serverRoot'
 import fetch from 'node-fetch'
+import { PlayerComponent } from 'common/components'
+import query from 'common/components/query'
+import { PlayerInfo } from 'common/types/server-requests'
 const require = createRequire(import.meta.url)
 
+type PlayerStateT = PlayerInfo & {
+	lives: number
+}
+
 export function registerApis(app: import('express').Express) {
 	let apiKeys: any = null
 	let botKey: any = null
 
 	const env = process.env.NODE_ENV || 'development'
 	if (env == 'development') {
-		console.log('running in dev mode, not activating api')
-		return
+		//console.log('running in dev mode, not activating api')
+		//return
 	}
 
 	try {
 		apiKeys = JSON.parse(process.env.API_KEYS || '')
 		botKey = process.env.BOT_KEY
-		console.log(apiKeys, botKey, process.env.BOT_URL)
 
 		// get info about games
 		app.get('/api/games', (req, res) => {
 			const apiKey = req.header('api-key')
 			if (apiKey) {
-				if (apiKeys?.keys.includes(apiKey)) {
+				if (apiKeys.includes(apiKey)) {
 					res.status(201).send(
 						JSON.stringify(
 							root.getGames().map((g: GameModel) => {
 								return {
 									createdTime: g.createdTime,
 									id: g.id,
 									code: g.code,
-									playerIds: g.getPlayerIds(),
-									playerNames: g.getPlayers().map((p) => p.name),
+									players: g.getPlayers().map((p) => {
+										const playerState: PlayerStateT = p.getPlayerInfo() as PlayerStateT
+										playerState.lives = g.components.filter(PlayerComponent, query.player.id(p.id))[0].lives
+										return playerState
```

Comment:
Are you able to do this?
```suggestion
									players: g.combinators.filter(PlayerComponent).map((player) => {
									     // The properties you need to send to the bot
									     lives: player.lives,
									}
```
It is not a good idea to monkeypatch objects. Please make a new dictionary instead.

Reply:
I was unaware I could do this, and agree this is much nicer